src/MediaManagerBundle/Resources/public/js/mediaManager.js

```diff
@@ -289,10 +295,12 @@
                     'id': encodeURIComponent(id)
                 },
                 error: function (responseData) {
+                    $.unblockUI();
```

Comment:
Just remembered $.unblockUI() is deprecated in 6.3. As of 6.3 we need to call: CHAMELEON.CORE.showProcessingModal() and CHAMELEON.CORE.hideProcessingModal(); So either we need seperate branches, or after the upmerge a change in 6.3/7.0.

Reply:
Added #408 for this.